src/components/EmptyPage/index.tsx

```diff
@@ -1,8 +1,8 @@
-import React from 'react';
+import React, {FC} from 'react';
 
 import './EmptyPage.scss';
 
-const EmptyPage = (): JSX.Element => {
+const EmptyPage: FC = () => {
```

Comment:
@amarjeet987 I actually like JSX.Element more, and in the future I do plan on removing the usage of `FC`, but for now, let's keep it as `FC` since that's what we use everywhere else